### ISSUE TITLE
Ivykis non-case-sensitive FSs

### ISSUE THREAD

**Issue by ngergo** [buytenh/ivykis#7]
I have just tried to compile ivykis on MacOSX using HFS without case-sensitive support. That's why the build has broken. The problem is in folder "man3" there are iv_fd_pump_init.3 and IV_FD_PUMP_INIT.3 files. The uppercase version hides the lowercase one. This problem happens the build to break. The same thing occurs on Windows platform. It would be great to be solved.

**Comment by opoplawski** [comment]
I'm seeing the same issue trying to build syslog-ng for MacOSX:
```
make[5]: *** No rule to make target `IV_FD_PUMP_INIT.3', needed by `all-am'.  Stop.
```

**Comment by buytenh** [comment]
Does this work?

```
diff --git a/Makefile.am b/Makefile.am
index 774da8c..f66f4c2 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,9 +1,6 @@
 SUBDIRS = src test test.mt
 if HAVE_POSIX
-SUBDIRS += contrib misc
-if !BUILD_ON_CYGWIN
-SUBDIRS += man3
-endif
+SUBDIRS += contrib man3 misc
 endif
 
 EXTRA_DIST = DEDICATION libivykis.posix.ver libivykis.win32.ver
diff --git a/configure.ac b/configure.ac
index 8cd7330..042bb02 100644
--- a/configure.ac
+++ b/configure.ac
@@ -279,7 +285,7 @@ AM_CONDITIONAL([HAVE_KQUEUE], [test x$ac_cv_func_kqueue = xyes])
 AM_CONDITIONAL([HAVE_PORT], [test x$ac_cv_func_port_create = xyes])
 
 # Other conditionals.
-AM_CONDITIONAL([BUILD_ON_CYGWIN], [test $build_os = cygwin])
+AM_CONDITIONAL([CASE_SENSITIVE_FS], [test $srcdir/COPYING -ef $srcdir/copying])
 AM_CONDITIONAL([HAVE_INOTIFY], [test x$ac_cv_func_inotify_init = xyes])
 AM_CONDITIONAL([HAVE_LINUX_NETFILTER_IPV4_H],
                [test x$ac_cv_header_linux_netfilter_ipv4_h = xyes])
diff --git a/man3/Makefile.am b/man3/Makefile.am
index 1508057..bc62d5e 100644
--- a/man3/Makefile.am
+++ b/man3/Makefile.am
@@ -14,7 +14,6 @@ man3_MANS     = iv_deinit.3                           \
                  iv_fd.3                               \
                  iv_fd_pump.3                          \
                  iv_fd_pump_destroy.3                  \
-                 IV_FD_PUMP_INIT.3                     \
                  iv_fd_pump_init.3                     \
                  iv_fd_pump_is_done.3                  \
                  iv_fd_pump_pump.3                     \
@@ -68,6 +67,10 @@ man3_MANS    = iv_deinit.3                           \
                  iv_work_pool_submit_work.3            \
                  ivykis.3
 
+if !CASE_SENSITIVE_FS
+man3_MANS      += IV_FD_PUMP_INIT.3
+endif
+
 if HAVE_INOTIFY
 man3_MANS      += iv_inotify.3                         \
                   IV_INOTIFY_INIT.3                    \
```

**Comment by opoplawski** [comment]
It looks like it does.

**Comment by buytenh** [comment]
Thanks!  @opoplawski Among the other Reported-by: tags, what email address can I use for you?

**Comment by opoplawski** [comment]
orion@nwra.com

**Comment by buytenh** [comment]
@ngergo What email address can I use for you, now that you are not with Balabit anymore?

**Comment by buytenh** [comment]
Fixed by 2c5e3437fd73, and released as part of v0.42.3.
